p2p-specs/canonical-mempools.md

```diff
@@ -0,0 +1,8 @@
+## The list of published canonical mempools are:
+
+| Chain Name | Chain ID | Mempool ID | Description (Link to Mempool file) |
+|---|---|---|---|
+| `Goerli` | 5 | `QmTmj4cizhWpEFCCqk5dP67yws7R2PPgCtb2bd2RgVPCbF` | https://ipfs.io/ipfs/QmTmj4cizhWpEFCCqk5dP67yws7R2PPgCtb2bd2RgVPCbF?filename=goerli_canonical_mempool.yaml
+| `Sepolia` | 11155111 | `QmdDwVFoEEcgv5qnaTB8ncnXGMnqrhnA5nYpRr4ouWe4AT` | https://ipfs.io/ipfs/QmdDwVFoEEcgv5qnaTB8ncnXGMnqrhnA5nYpRr4ouWe4AT?filename=sepolia_canonical_mempool.yaml
+| `Mumbai` | 80001 | `QmQfRyE9iVTBqZ17hPSP4tuMzaez83Y5wD874ymyRtj9VE` | https://ipfs.io/ipfs/QmQfRyE9iVTBqZ17hPSP4tuMzaez83Y5wD874ymyRtj9VE?filename=mumbai_canonical_mempool.yaml
+
```

Comment:
1. we should reference the source documents (in this repo) of the mempools (its in another pr)
2. should have a clear definition of what the ID is. (that is, how to take a clear-text, and calculate the CID of it)
   should copy/reference a definition - but I couldn't find it in the IPFS spec.
   I _think_ it is something like `"Qm"+ base58(keccak256(text))`, but not exactly. even "kubo" tool is inconsistent, in that `kubo < file` gives different ID from `kubo file` ...